.github/workflows/docker_release.yml

```diff
@@ -19,12 +19,28 @@ jobs:
       - name: Set env
         id: set_tag_version
         run: |
-          if [ "${{ github.event.client_payload.tagversion }}" != "" ]; then
-          echo ::set-output name=tagversion::${{ github.event.client_payload.tagversion }}
+          #Checks for when a valid tag is manually pushed
+          if [[ $GITHUB_REF != "" && "${{ github.event.client_payload.tagversion }}" == "" ]]; then
+          echo ::set-output name=tagversion::$(egrep -o '(v[0-9]+\.[0-9]+\.[0-9]+)(-.*)?' <<< $GITHUB_REF)
           else
-          echo ::set-output name=tagversion::$(echo $GITHUB_REF | cut -d / -f 3)
+          #Runs when a tag is triggered automatically
+          echo ::set-output name=tagversion::${{ github.event.client_payload.tagversion }}
           fi
           echo $tagversion
+      - name: Check Tag Version
+        run: |
+          if [[ "${{ steps.set_tag_version.outputs.tagversion }}" == ""  ]]; then
+          echo "::error::Incorrect tag provided, please provide a tag with the correct format and try again"
+          exit 1
+          fi
+      - name: Docker tag version
+        id: set_docker_tag_version
+        run: |
+          tagversion=${{ steps.set_tag_version.outputs.tagversion}}
+          res=
+          echo ::set-output name=dockertag::$(sed 's/^v//' <<< "$tagversion")
```

Comment:
This removes the v added during the tagging process in order to get the docker version